packages/router/src/router.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { useCallback, useEffect, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { RouterContext, history } from './context';
+import { getLinkElement, hasLocalTarget, parseLocation } from './util';
+
+export const Router = ( { children, allowedRoutes = /.*/i } ) => {
```

Comment:
is `allowedRoutes` a single regex for all? Could/should we turn this into an array?

Reply:
I guess we could... that's a good point.  
Let's maybe open up an issue for that or something just to get the initial things merged so it can easily be tested when we alter it.